game/templates/game/landing.html

```
@@ -128,17 +128,19 @@ <h1 class="hero-title">
+                we deliver a competitive and seamless experience for players of all levels.
             </p>
             <p>
-                The platform follows a hybrid architecture where Django and Python handle application flow, authentication, and game orchestration, while a high-performance C++ minimax engine powers intelligent AI gameplay.
-            </p>
-            <p>
-                With responsive gameplay, statistics tracking, rulebook access, and AI-driven interactions, Checkora provides both a practical learning environment and a smooth competitive chess experience for players across skill levels.
+              <strong>Join the Community:</strong> Checkora is built by contributors like you. We welcome developers of all skill 
+              levels to help us improve our UI, optimize our engine, and build new features. 
+              <a href="https://discord.gg/DvW3xVXw8g" target="_blank">Join our Discord</a> to get started!            </p>
+              With responsive gameplay, statistics tracking, rulebook access, and AI-driven interactions, Checkora provides both a practical learning environment and a smooth competitive chess experience for players across skill levels.
             </p>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </p> ]

(tag-pair)